git-publish

```diff
@@ -365,7 +366,7 @@ def tag_name_staging(topic):
 
 def tag_name_pull_request(topic):
     '''Build a pull request tag name from a topic name'''
-    return '%s-pull-request' % topic
+    return '%s-%s' % (topic, time.strftime('%Y%m%d'))
```

Comment:
This is not safe if git-publish runs across midnight (23:59 -> 00:00) since tag_name_pull_request() is called multiple times and may generate different return values.

Please change this to a global variable so the string is only generated once.